src/components/base/BaseCard.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <v-sheet>
+    <v-card v-bind="$attrs" :class="{ 'c-card': !embedded }" :flat="embedded">
```

Comment:
I read that [v-sheet is the base for v-card](https://vuetifyjs.com/en/components/sheets/), so do we want use `v-card` here inside `v-sheet`? Is using `div` making more sense?

Reply:
This is an unnecessary label, it should delete all directly.

Reply:
> This is an unnecessary label, it should delete all directly.

What do you mean by "this", and "all"?

Reply:
`v-sheet` could extend theme provider and write some vuetify props, I suggest use `v-sheet` to replace `div` in most cases.

Reply:
https://vuetifyjs.com/en/components/sheets/

Reply:
Oh, my earlier comment is about the `v-card` inside the `v-sheet`. I was thinking whether the use of `v-card` is necessary.

Reply:
> Oh, my earlier comment is about the v-card inside the v-sheet. I was thinking whether the use of v-card is necessary.

Yes, but #45 should solve that repeated cards are used superimposed, I noticed a lot of unnecessary nesting of 'v-card'